lib/service/service.go

```diff
@@ -3199,6 +3201,26 @@ func (process *TeleportProcess) initProxyEndpoint(conn *Connector) error {
 		}
 	}
 
+	var grpcServer *grpc.Server
+	if alpnRouter != nil {
```

Comment:
No ALPN == No IAM node registering, right?

I guess that is fair, but let's make sure we mention it the public docs.

Reply:
The ALPN router is only disabled in some tests, this works with both `ProxyListenerMode_Multiplex` and `ProxyListenerMode_Separate`